tests/commit.bats

```diff
@@ -96,3 +96,26 @@ load helpers
   cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid
 }
+
+@test "commit with name" {
+  buildah from --name nginxc nginx
+  run_buildah --log-level=error commit --signature-policy ${TESTSDIR}/policy.json nginxc commitbyname/nginxbyname
+  cid=$(buildah from localhost/commitbyname/nginxbyname)
+  run_buildah inspect $cid
+  buildah rm nginxc $cid
+  buildah rmi commitbyname/nginxbyname nginx
```

Comment:
See above comments about rmi. (In this case `rmi nginxbyname` is good but not `rmi your-base-image`).